core/services/pg/q.go

```diff
@@ -123,13 +123,13 @@ type Q struct {
 	QueryTimeout time.Duration
 }
 
-func NewQ(db *sqlx.DB, logger logger.Logger, config QConfig, qopts ...QOpt) (q Q) {
+func NewQ(db *sqlx.DB, lggr logger.Logger, config QConfig, qopts ...QOpt) (q Q) {
 	for _, opt := range qopts {
 		opt(&q)
 	}
 
 	q.db = db
-	q.logger = logger.Helper(2)
+	q.logger = logger.Helper(lggr, 2)
```

Comment:
while you are changing this, can you add a comment about the mythical importance of 2?

Reply:
I don't exactly follow. Presumably this cases needs to skip two levels in the stack. Is it important to elaborate more than that?
Would a comment still be necessary if the method were named differently?

Reply:
Updated